<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bollo - Pre-alpha [Codename: Baguette]</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Estado de panaderia</string>
      </property>
      <widget class="QLabel" name="mixingIcon">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="firstFermentIcon">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="doughDivisionIcon">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="doughFormingIcon">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="finalFermentIcon">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="bakingIcon">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="shippedIcon">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="onSaleIcon">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/assets/pending-icon.png</pixmap>
       </property>
      </widget>
      <widget class="QProgressBar" name="firstFermentBar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="doughDivisionBar">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="doughFormingBar">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="finalFermentBar">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="bakingBar">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="onSaleBar">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="shippedBar">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>80</y>
         <width>81</width>
         <height>6</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: #808080;
	border-radius: 2px;
 }

QProgressBar::chunk {
    background-color: #5BC614;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title1">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Mezcla de ingredientes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title2">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>110</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Fermentacion primaria</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title3">
       <property name="geometry">
        <rect>
         <x>265</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Division de la masa</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title4">
       <property name="geometry">
        <rect>
         <x>394</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Formando panes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title5">
       <property name="geometry">
        <rect>
         <x>513</x>
         <y>110</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Fermentacion final</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title6">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>110</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Horneo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title7">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>110</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Venta</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title8">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>110</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Distribucion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="incrementBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Increment</string>
       </property>
      </widget>
      <widget class="QLabel" name="statusLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Estado:  ----</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeRemainingLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tiempo restante:       --:--</string>
       </property>
      </widget>
      <widget class="QLabel" name="elapsedTimeLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tiempo transcurrido:  --:--</string>
       </property>
      </widget>
      <widget class="QLabel" name="createdBreadsLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Panes creados: ----</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewBakery"/>
    <addaction name="actionSignOut"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Temas</string>
     </property>
     <addaction name="actionThemeLight"/>
     <addaction name="actionThemeDark"/>
    </widget>
    <addaction name="actionEditBakerys"/>
    <addaction name="menuThemes"/>
    <addaction name="actionPreferencesPanel"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLogBrowser"/>
    <addaction name="actionGraphViewer"/>
    <addaction name="actionTelegram"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBakeryList"/>
   <addaction name="actionShowGraphs"/>
   <addaction name="actionBakeryLog"/>
   <addaction name="separator"/>
   <addaction name="actionContinueProcess"/>
   <addaction name="actionPauseProcess"/>
   <addaction name="actionCancelProcess"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionBakeryList">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/assets/bread-icon.png</normaloff>:/images/assets/bread-icon.png</iconset>
   </property>
   <property name="text">
    <string>bakeryList</string>
   </property>
   <property name="toolTip">
    <string>Lista de panaderias</string>
   </property>
  </action>
  <action name="actionShowGraphs">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/assets/graph-icon.png</normaloff>:/images/assets/graph-icon.png</iconset>
   </property>
   <property name="text">
    <string>showGraphs</string>
   </property>
   <property name="toolTip">
    <string>Estadisticas y Graficos de la panaderia</string>
   </property>
  </action>
  <action name="actionContinueProcess">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/assets/continue-icon.png</normaloff>:/images/assets/continue-icon.png</iconset>
   </property>
   <property name="text">
    <string>continueProcess</string>
   </property>
   <property name="toolTip">
    <string>Reanuda el proceso de creacion del pan.</string>
   </property>
  </action>
  <action name="actionPauseProcess">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/assets/pause-icon.png</normaloff>:/images/assets/pause-icon.png</iconset>
   </property>
   <property name="text">
    <string>pauseProcess</string>
   </property>
   <property name="toolTip">
    <string>Detiene el proceso de creacion del pan.</string>
   </property>
  </action>
  <action name="actionCancelProcess">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/assets/cancel-icon.png</normaloff>:/images/assets/cancel-icon.png</iconset>
   </property>
   <property name="text">
    <string>cancelProcess</string>
   </property>
   <property name="toolTip">
    <string>Cancela el proceso de creacion del pan (Irreversible)</string>
   </property>
  </action>
  <action name="actionBakeryLog">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/assets/log-icon.png</normaloff>:/images/assets/log-icon.png</iconset>
   </property>
   <property name="text">
    <string>bakeryLog</string>
   </property>
   <property name="toolTip">
    <string>Bitacora de eventos de la panaderia</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Informacion de la aplicacion</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Guia de ayuda</string>
   </property>
  </action>
  <action name="actionNewBakery">
   <property name="text">
    <string>Nueva panaderia</string>
   </property>
   <property name="toolTip">
    <string>Agregar nueva panaderia</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Cerrar la aplicacion</string>
   </property>
  </action>
  <action name="actionSignOut">
   <property name="text">
    <string>Cerrar sesion</string>
   </property>
  </action>
  <action name="actionLogBrowser">
   <property name="text">
    <string>Navegador de bitacoras</string>
   </property>
   <property name="toolTip">
    <string>Explorar las bitacoras del sistema</string>
   </property>
  </action>
  <action name="actionGraphViewer">
   <property name="text">
    <string>Visor de graficos</string>
   </property>
   <property name="toolTip">
    <string>Graficos estadisticos de las panaderias</string>
   </property>
  </action>
  <action name="actionTelegram">
   <property name="text">
    <string>Telegram</string>
   </property>
   <property name="toolTip">
    <string>Ajustes de integracion con Telegram</string>
   </property>
  </action>
  <action name="actionThemeLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
   <property name="toolTip">
    <string>Tema por defecto (Light)</string>
   </property>
  </action>
  <action name="actionThemeDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
   <property name="toolTip">
    <string>Tema oscuro (Dark)</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>Cambiar contrasena</string>
   </property>
   <property name="toolTip">
    <string>Cambiar contrasena de usuario</string>
   </property>
  </action>
  <action name="actionEditBakerys">
   <property name="text">
    <string>Editar panaderias</string>
   </property>
  </action>
  <action name="actionPreferencesPanel">
   <property name="text">
    <string>Preferencias</string>
   </property>
   <property name="toolTip">
    <string>Preferencias de la aplicacion</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
